Makefile

```diff
@@ -76,3 +77,11 @@ wtr:
 
 gdb-wtr: wtr
 	gdb wtr
+
+artifact: libcarrow.a gen artifact-linux-x86_64
+
+artifact-linux-x86_64:
```

Comment:
IMO this rule should depend on `libcarrow.a` and not `artifact` rule

Reply:
Good idea